static/app/components/commitRow.tsx

```diff
@@ -153,6 +148,8 @@ const Meta = styled(TextOverflow)`
   color: ${p => p.theme.subText};
 `;
 
-export default styled(CommitRow)`
+const StyledCommitRow = styled(CommitRow)`
```

Comment:
does this work? the `CommitRow` component does not pass down any className

Reply:
would it be better if we could style `PanelItem` and remove `export {StyledCommitRow as CommitRow};`?

Reply:
@priscilawebdev Ouf, nice catch. This seems to have "just worked" before and escaped TS. I'll add it back!

Reply:
> would it be better if we could style PanelItem and remove export {StyledCommitRow as CommitRow};?

I'm not sure I know enough to give an opinion, but it seems that we are using `StyledCommitRow` to center the items in our row... I usually try and avoid setting things like alignment, margin and padding rules on components directly and rather allow whoever is using the components to define their own alignments/margins. It's more flexible to do something like this
```tsx
<wrapper style={{alignItems: "center"}}>
<CommitRow/>
</wrapper>
```
It's a bit more tedious, but less opinionated. That said, I wouldnt use this as a rule because maybe we always want align-items center to apply here, in which case I think it's fine (and the right thing to do).

Reply:
I understand. If I remember well, I think there were already some discussions in our Frontend TSC meetings on this topic, but to this day we still haven't managed to reach a consensus. Maybe we should discuss this again?

Regarding the inline style, we tend to avoid them and only use them if it's really necessary